### PR TITLE
Fixed empty info and PUBLIC_URL of modernizr

### PR DESCRIPTION
**Pull request by Liljenmaa**

### CODE REVIEW

**Review by Shandorin — APPROVED** [review]
Working as intended

File: front/src/weekview/Weekview.js [hunk 1]
```diff
@@ -320,7 +320,8 @@ class Weekview extends Component {
       info = false;
       if (this.state.paivat[j].tracks) {
         this.state.paivat[j].tracks.forEach((track) => {
-          if (track.notice !== null) {
+          if (track.notice !== null && track.notice !== "") {
+            Attention = track.notice;
```
Comment (TuoLuo994) [inline front/src/weekview/Weekview.js:324]: Line 324:13:  'Attention' is not defined  no-undef

Project doesn't run.

**Review by TuoLuo994 — CHANGES_REQUESTED** [review]

File: front/src/weekview/Weekview.js [hunk 2]
```diff
@@ -320,7 +320,8 @@ class Weekview extends Component {
       info = false;
       if (this.state.paivat[j].tracks) {
         this.state.paivat[j].tracks.forEach((track) => {
-          if (track.notice !== null) {
+          if (track.notice !== null && track.notice !== "") {
```
Comment (TuoLuo994) [inline front/src/weekview/Weekview.js:323]: Linting:  Strings must use singlequote

**Review by Shandorin — CHANGES_REQUESTED** [review]
Just added the fixed part to my locak so missed Attention part. that needs to be fixed

File: front/src/weekview/Weekview.js (hunk 1 above)
Comment (Liljenmaa) [reply front/src/weekview/Weekview.js]: Interesting, I think there was a merge conflict with an old version of develop which dazzled me with what wasn't supposed to actually be there. Need to be careful in the future with this. Fixed.

**Review by Shandorin — APPROVED** [review]

**Review by TuoLuo994 — APPROVED** [review]
Looks good @Shandorin please take a look and merge if everything ok.

Edit: never mind, you were 20 seconds faster than me

**Pull request merged**